utils/config/daos_server.yml

```diff
@@ -167,12 +167,16 @@
 #socket_dir: ./.daos/daos_server
 #
 #
-## Number of hugepages to allocate for use by NVMe SSDs
-#
-## Specifies the number (not size) of hugepages to allocate for use by NVMe
-## through SPDK. This indicates the number to be used for each spawned
-## I/O Engines, so the total will be this number * number of I/O Engines.
-## Default system hugepage size will be used.
+## Number of hugepages to allocate for DMA buffer memory
+#
+## Specifies the number (not size) of hugepages to allocate for use by DMA
+## buffer memory, this is mandatory with NVME SSD configured and optional
+## without SSDs configured. This indicates the number to be used for each
+## spawned I/O Engines, so the total will be this number * number of I/O Engines.
+## Default system hugepage size will be used. For the best performance,
+## It requires at least 1GiB memory size per target, so for one engine with
+## 8 targets, 4096 is minimum. When NVMe isn't configured, BIO will just
```

Comment:
'BIO' isn't something user can understand.

Reply:
oops, missed this comments.

Reply:
sorry, missed this comments, this will be addressed together in case test hit failures or new comments.